v2/tools/generator/internal/codegen/pipeline/report_resource_versions.go

```diff
@@ -77,6 +77,11 @@ func (r *ResourceVersionsReport) WriteTo(outputPath string, samplesURL string) e
 
 // WriteToBuffer creates the report in the provided buffer
 func (r *ResourceVersionsReport) WriteToBuffer(buffer *strings.Builder, samplesURL string) {
+
+	buffer.WriteString("# Supported Resources\n\n")
+	buffer.WriteString("This page lists all resources currently supported by Azure Service Operator, grouped by the originating ARM service.\n\n")
```

Comment:
Should we clarify what exactly `current` means? In this context it's actually what is supported in `master` rather than at a particular release - so it's possible there are items in this list that aren't really usable yet (because we haven't released since they were added?)